TI_Practice1/Validator.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Globalization;
+using System.Text.RegularExpressions;
+
+namespace Practice1
+{
+    class Validator
+    {
+        private int _attempts;
+
+        public Validator(int attempts)
+        {
+            _attempts = attempts;
+        }
+
+        public double IsNumber()
```

Comment:
method name that starts with "Is.." should return bool value.

I suggest you name it something like GetValueFromConsole() or anything else that reflects the actions you do inside

Reply:
DONE